packages/sdk-js/src/types/growthbook.ts

```diff
@@ -398,11 +399,13 @@ export type InitOptions = {
   skipCache?: boolean;
   payload?: FeatureApiResponse;
   streaming?: boolean;
+  cacheSettings?: CacheSettings;
 };
 
 export type InitSyncOptions = {
   payload: FeatureApiResponse;
   streaming?: boolean;
+  cacheSettings?: CacheSettings;
```

Comment:
I don't think we need this for `initSync`, since that method doesn't use the feature repository at all.